packages/core/src/balance/Balance.chain.ts

```diff
@@ -124,15 +124,15 @@ export async function listenToBalanceChanges(
  * const address = ...
  * const amount: BN = new BN(42)
  * const blockchain = await sdk.getConnectionOrConnect()
- * sdk.Balance.makeTransfer(address, amount, 0) //
+ * sdk.Balance.getMakeTransferTx(address, amount, 0) //
  *   .then(tx => BlockchainUtils.signAndSendTx(tx, identity))
  *   .then(() => console.log('Successfully transferred ${amount.toNumber()} tokens'))
  *   .catch(err => {
  *     console.log('Transfer failed')
  *   })
  * ```
  */
-export async function makeTransfer(
+export async function getMakeTransferTx(
```

Comment:
S: I think `getTransferTx` would be even better here.

Reply:
There are more places, where it is still `getMakeTransferTx` please search through your whole codebase.